### ISSUE TITLE
Request: Support added for PNP

### ISSUE THREAD

**Issue by box293** [Griesbacher/histou#22]
It would be great if PNP could be added as a supported Datasource.

https://github.com/sni/grafana-pnp-datasource
https://github.com/lingej/pnp-metrics-api

**Comment by Griesbacher** [comment]
Yes that would be possible, there exists a prove of concept for Elasticsearch already and likewise the PNP datasource could work.

Pull Requests are welcome! :)

**Comment by box293** [comment]
Yeah I was having a dabble with the code but got a bit lost ... my coding skills are the problem lol

I'll see if I can get something working.
